spec/functional/plausibility_spec.rb

```diff
@@ -579,7 +579,7 @@ def self.it_is_correct_for(country_name, options={})
       it_is_correct_for 'Faroe Islands', :samples => '+298  969 597'
       it_is_correct_for 'Fiji (Republic of)', :samples => '+679  998 2441'
       it_is_correct_for 'French Guiana (French Department of)', :samples => '+594 594 123 456'
-      it_is_correct_for "French Polynesia (Territoire français d'outre-mer)", :samples => '+689  872 784'
+      it_is_correct_for "French Polynesia (Territoire français d'outre-mer)", :samples => '+689  872 784 00'
```

Comment:
Should this not be in a 2-2-2-2 grouping? (I mean, it passes, but for not confusing readers)